compiler/src/dmd/mtype.d

```diff
@@ -4871,9 +4871,18 @@ extern (C++) final class TypeFunction : TypeNext
     }
 
     /// Returns: whether `this` function type has the same attributes (`@safe`,...) as `other`
-    bool attributesEqual(const scope TypeFunction other) const pure nothrow @safe @nogc
-    {
-        return this.trust == other.trust &&
+    extern (D) bool attributesEqual(const scope TypeFunction other, bool trustSystemEqualsDefault = false) const pure nothrow @safe @nogc
```

Comment:
I think the default should be `true` as the `false` is only used in one place, isn't it ?

Reply:
It's two places: semantic2.d and opover.d.

The caller score for true vs. false is 2-2.  The semantic2.d calls it twice, first with `false`, then second with `true` to catch the bug and issue deprecation.

Reply:
I'll flip it over in master after merge_stable is done.

https://github.com/dlang/dmd/pull/14821